.github/workflows/ci-build-image.yml

```diff
@@ -0,0 +1,75 @@
+name: CI Build / Push Image 
+on:
+  repository_dispatch:
+    types: [build-image-command]
+
+env:
+  REGISTRY:  ghcr.io
+  REPOSITORY: ${{ github.event.client_payload.github.payload.repository.full_name }}
+  DEFAULT_TAG: ${{ github.event.client_payload.github.payload.issue.number }}
+  FULL_NAME: fabric8-analytics-stack-report-ui
+ 
+
+jobs:
+  Build-push-image:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          git fetch origin +refs/pull/${{ github.event.client_payload.github.payload.issue.number }}/merge
+          git checkout FETCH_HEAD
+      - name: send comment 
+        env:
+          URL: ${{ github.event.client_payload.github.payload.issue.comments_url }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          curl \
+            -X POST \
+            $URL \
+            -H "Content-Type: application/json" \
+            -H "Authorization: token $GITHUB_TOKEN" \
+            --data '{ "body": ":test_tube: [CI has Started Image Build]( https://github.com/'"$GITHUB_REPOSITORY"'/actions/runs/'"$GITHUB_RUN_ID"')" }'
```

Comment:
Don't we have any action to comment on PR? e.g. https://github.com/marketplace/actions/comment-on-github

Reply:
we have it can directly comment on PR but in our case its an issue comment / slash command so have to use the API